app/views/tenants/dashboard/show.html.erb

```diff
@@ -0,0 +1,27 @@
+<%= render "shared/back_button" %>
+<div id="admin_header">
+  <h1><%= @tenant.organization %>'s Dashboard</h1>
+</div>
+
+<div id="projects">
+  <table class='table table-bordered'>
+    <thead>
+      <tr class="warning">
+        <th data-field='Project'> Project </th>
+        <th data-field='Price'>Total Cost</th>
```

Comment:
Total cost of the project. I know the column is named "Price", but as a borrower, I would think of it as "total cost".